src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/GetInductionTest.kt

```diff
@@ -101,16 +100,13 @@ class GetInductionTest : IntegrationTestBase() {
       .wasUpdatedByDisplayName("Alex Smith")
       .wasCreatedAtPrison("BXI")
       .wasUpdatedAtPrison("BXI")
-
-    with(actual) {
```

Comment:
'actual` and therefore all of it's child properties could be null (as far as kotlin is concerned), hence the warnings.
We already have a custom assertJ assertion class for `InductionResponse` (ie. `actual`) so I have added methods to it to assert the child properties.